macos/FluentUI/Button/Button.swift

```diff
@@ -175,304 +271,355 @@ open class Button: NSButton {
+		}
+		set {
+			borderColorRest = newValue
+			borderColorPressed = newValue == .clear ? newValue : newValue?.withSystemEffect(.pressed)
+			borderColorDisabled = newValue?.withSystemEffect(.disabled)
 		}
 	}
 
-	private static var defaultPrimaryColor: NSColor {
-		return .controlAccentColor
-	}
-	
-	
-	class FluentButtonCell : NSButtonCell {
-		override func imageRect(forBounds rect: NSRect) -> NSRect {
-			let isRTL = NSApp.userInterfaceLayoutDirection == .rightToLeft
-			let imageRect = super.imageRect(forBounds: rect)
-			let titleSize = title.size(withAttributes: [.font: font as Any])
-			let imageSize = image?.size ?? NSZeroSize
-			
-			var x = imageRect.origin.x
-			var y = imageRect.origin.y
-			var width = imageRect.size.width
-			var height = imageRect.size.height
-			
-			switch imagePosition {
-			case .noImage:
-				x = 0
-				y = 0
-				width = 0
-				height = 0
-				break
-			case .imageOnly:
-				x = Button.horizontalEdgePadding
-				y = Button.verticalEdgePadding
-				width = rect.width - (Button.horizontalEdgePadding * 2)
-				height = rect.height - (Button.verticalEdgePadding * 2)
-			case .imageLeft:
-				x = Button.horizontalEdgePadding
-				y = Button.verticalEdgePadding
-				width = imageSize.width
-				height = imageSize.height
-				break
-			case .imageRight:
-				x = titleSize.width + Button.titleAndImageRectInterspacing + Button.horizontalEdgePadding
-				y = Button.verticalEdgePadding
-				width = imageSize.width
-				height = imageSize.height
-				break
-			case .imageBelow:
-				x = (rect.width - imageSize.width) / 2
-				y = Button.verticalEdgePadding + titleSize.height + Button.titleAndImageRectInterspacing
-				width = imageSize.width
-				height = imageSize.height
-				break
-			case .imageAbove:
-				x = (rect.width - imageSize.width) / 2
-				y = Button.verticalEdgePadding
-				width = imageSize.width
-				height = imageSize.height
-				break
-			case .imageOverlaps:
-				x = Button.horizontalEdgePadding
-				y = Button.verticalEdgePadding
-				width = rect.width - (Button.horizontalEdgePadding * 2)
-				height = rect.height - (Button.verticalEdgePadding * 2)
-				break;
-			case .imageLeading:
-				x = isRTL ? titleSize.width + Button.titleAndImageRectInterspacing + Button.horizontalEdgePadding : Button.horizontalEdgePadding
-				y = Button.verticalEdgePadding
-				width = imageSize.width
-				height = imageSize.height
-				break
-			case .imageTrailing:
-				x = isRTL ? Button.horizontalEdgePadding : titleSize.width + Button.titleAndImageRectInterspacing + Button.horizontalEdgePadding
-				y = Button.verticalEdgePadding
-				width = imageSize.width
-				height = imageSize.height
+	/// This color is used for the background in the primary style, the pressed background in the secondary
+	/// style, and the content tint (i.e. foreground text/image) color in the borderless style.  It is not used
+	/// for the acrylic style.
+	@objc public var accentColor: NSColor? = Colors.primary
```

Comment:
What happens if someone sets both this and contentTintColor on a borderless button?

Reply:
Last one set wins.

Reply:
Update: with the latest changes we're no longer respecting contentTintColor.  If the client attempts to set it on a button, we'll clobber it with contentTintColor{Rest,Pressed,Disabled} the next time the pressed or enabled state changes.